src/lib/Metadata/Metadata.js

```diff
@@ -1,134 +1,113 @@
-/*!
- * Copyright(c) ConsenSys Software Inc.
- * Copyright(c) https://consensys.net/
- * MIT Licensed
- */
-
-import { readFileSync } from 'fs';
-
-import IPFS from '../../services/ipfsService.js';
-import { ERROR_LOG, errorLogger } from '../error/handler.js';
+import { errorLogger, ERROR_LOG } from '../error/handler.js';
+import {
+  contractMetadataSchema,
+  freeMetadataSchema,
+  tokenMetadataSchema,
+} from './metadata.schema.js';
 
 export default class Metadata {
-  IPFS_BASE_URL = 'https://ipfs.io/ipfs/';
+  /**
+   * Create token metadata
+   * @param {string} name Name of the token
+   * @param {string} description description of the token
+   * @param {string} image tokenURI
+   * (link to a JSON file describing the token's metadata)
+   * @param {Array} attributes The attributes of the token
+   * @param {string} animation_url Link to the animation such as music, video
+   * @returns string
+   */
+  static tokenLevelMetadata({ name, description, image, attributes, animation_url = null }) {
```

Comment:
Wasn't it supposed to follow OpenSea standard ? https://docs.opensea.io/docs/metadata-standards#metadata-structure